go/src/net/ConnectionPool.go

```diff
@@ -0,0 +1,247 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package nebulaNet
+
+import (
+	"container/list"
+	"errors"
+	"log"
+	"sync"
+
+	graph "github.com/vesoft-inc/nebula-clients/go/nebula/graph"
+	conf "github.com/vesoft-inc/nebula-clients/go/src/conf"
+	data "github.com/vesoft-inc/nebula-clients/go/src/data"
+)
+
+var defaultPoolConfigs = conf.GetDefaultConf()
+
+type RespData struct {
+	Resp *graph.ExecutionResponse
+	Err  error
+}
+
+type ConnectionPool struct {
+	idleConnectionQueue   list.List
+	activeConnectionQueue list.List
+	addresses             []*data.HostAddress
+	conf                  *conf.PoolConfig
+	loadBalancer          *LoadBalancer
+	ConnectionInUse       map[*Connection]bool
```

Comment:
the var name format 

your connection can be deleted, but the ConnectionInUse never delete the connection